backend/tracim_backend/lib/core/user.py

```diff
@@ -887,7 +889,8 @@ def execute_updated_user_actions(self, user: User) -> None:
         # event mecanism is ready, see https://github.com/tracim/tracim/issues/1487
         # event on_updated_user should start hook use by agenda  app code.
 
-        if self._config.CALDAV__ENABLED:
+        app_lib = ApplicationApi(app_list=app_list)
```

Comment:
This kind of code, will be moved to app specific code as soo as new event are added for this. see https://github.com/tracim/tracim/issues/1487